package.json

```diff
@@ -95,7 +95,7 @@
     "vscode-builtin-log": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/log-1.39.1-prel.vsix",
     "vscode-builtin-lua": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/lua-1.39.1-prel.vsix",
     "vscode-builtin-make": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/make-1.39.1-prel.vsix",
-    "vscode-builtin-markdown": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/markdown-1.39.1-prel.vsix",
+    "vscode-builtin-markdown": "https://registry.npmjs.org/@theia/vscode-builtin-markdown/-/vscode-builtin-markdown-0.3.0-next.3c6f6af734.tgz",
```

Comment:
@marcdumais-work @vince-fugnitto we probably need a new release from latest VS Code for built-ins, 1.39 does not render some embedded languages properly, like shell scripts

Reply:
nvm it is not related to a version, it is a bug in our code only